app/helpers/playbook/pb_kit_helper.rb

```diff
@@ -7,47 +7,25 @@
 module Playbook
   module PbKitHelper
     def pb_rails(kit, props: {}, &block)
-      render_rails(kit, props, &block)
+      kit = build_view_model(kit, props, &block)
+      render(partial: kit, as: :object)
     end
 
     def pb_react(kit, props: {}, options: {})
-      render_react(kit, props, options)
+      ::Webpacker::React::Component.new(kit.camelize).render(props, options)
     end
 
   private
 
-    def render_rails(kit, props, &block)
-      props = defined?(props) && !props.nil? ? props : {}
-      kit_class_obj = get_class_name(kit)
-      original_value = ActionView::Base.prefix_partial_path_with_controller_namespace
-      ActionView::Base.prefix_partial_path_with_controller_namespace = true
-      render(partial: kit_class_obj.new(**props, &block), as: :object)
-    rescue ActionView::MissingTemplate
```

Comment:
Would be great to see this all go. You'll just have to absolutely verify both parent and child partials render properly in `nitro-web`.